README.md

```diff
@@ -41,20 +41,19 @@ Instead, try to be more descriptive, for example:
 - For consistency, use **bold** to highlight UI text when dealing with UI-focused documentation (for example, Apify Console).
 - For consistency, use _italics_ to emphasize text.
 - For inline `code` examples, use **back-ticks** (\` \`).
-- For multi-line code examples, use code fences and specify the language. Preferably, specify the title as well.
-  - Within platform documentation, always use [Code tabs](README.md#code-tabs) and declare the language.
+- For multi-line code examples, use code fences and specify the language. Preferably, specify the title as well. Within platform documentation, always use [Code tabs](README.md#code-tabs) and declare the language.
 
-    ```markdown
-    \`\`\`js showLineNumbers title='foo.js'
+  ```markdown
+  \`\`\`js showLineNumbers title='foo.js'
 
-    const docsAreCool = require('coolDocs'); <br/>
-    ...<br/>
-    return docsAreCool;<br/>
+  const docsAreCool = require('coolDocs'); <br/>
+  ...<br/>
+  return docsAreCool;<br/>
 
-    \`\`\`
-    ```
+  \`\`\`
```

Comment:
This renders with the backslashes, but I didn't figure out how to do it in a correct way, so I left it as it was.

Reply:
https://stackoverflow.com/questions/42319711/fenced-code-block-inside-another-fenced-code-block

Reply:
I see I gave up too early! With all the AI around one would forget about the good old StackOverflow 😀 Thanks! Interesting solution, lol, adding more backticks. And it seems to work! At least in VS Code Markdown preview.